lib/utils/authorizeSilently.ts

```diff
@@ -0,0 +1,52 @@
+import { SilentAuthorizationError } from "../errors";
+import { TimeoutID } from "../types";
+
+const SERVER_MESSAGE_TIMEOUT = 10000;
+
+export const authorizeSilently = (uri: string, origin: string) => {
+  return new Promise((resolve, reject) => {
+    let timeoutId: TimeoutID;
+    const iframe = document.createElement("iframe");
+    iframe.style.display = "none";
+    document.body.appendChild(iframe);
+
+    const cleanup = () => {
+      window.removeEventListener("message", onMessageHandler);
+      if (iframe && iframe.parentNode) {
```

Comment:
<img width="455" alt="Screenshot 2019-11-05 at 15 51 16" src="https://user-images.githubusercontent.com/3253522/68283768-752c3e80-007c-11ea-812b-0fbe52ddfa8d.png">

🤷‍♂

Reply:
`document.body.removeChild(iframe);`?

Reply:
I was a bit reluctant to do that because if the SDK is used in a context where other `iframe` are attached to the body by any other code it could be problematic I think 🤔